File: Python/models/borrow_record.py
class BorrowRecord:
    def __init__(self):
        self.borrow_id = 0
        self.reader_id = 0
        self.book_id = 0
        self.reader_name = ""
        self.book_name = ""
        self.borrow_date = None
        self.due_date = None
        self.return_date = None
        self.status = 0
        self.late_fee = 0.0
        self.remark = ""

File: BM.Model/BorrowRecord.cs
using System;

namespace BookManagement.Model
{
    public class BorrowRecord
    {
        public int BorrowId { get; set; }
        public int ReaderId { get; set; }
        public int BookId { get; set; }
        public string ReaderName { get; set; }
        public string BookName { get; set; }
        public DateTime BorrowDate { get; set; }
        public DateTime DueDate { get; set; }
        public DateTime? ReturnDate { get; set; }
        public int Status { get; set; }
    }
}